config/routes.rb

```diff
@@ -2775,7 +2775,9 @@
   API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/ unless defined?(API_VERSION_REGEX)
 
   # OPTIONS requests for REST API pre-flight checks
-  match '/api/*path' => 'api/base#handle_options_request', :via => [:options]
+  match '/api' => 'api/base#handle_options_request', :via => :options
+  match '/api/arbitration_rules' => 'api/arbitration_rules#options', :via => :options
+  match '/api(/*path)' => 'api/base#options', :via => :options
```

Comment:
I didn't expect arbitration_rules to be separately matched,  actually would prefer extending API_ACTIONS below for :options and leveraging action_for_collection and driving this via method sets (i.e. &gpppdo) or even collection option (i.e. :support_option) in api.yml.   @imtayadeway thoughts ?